router/router.go

```diff
@@ -176,6 +176,10 @@ func Create(db *database.GormDatabase, vInfo *model.VersionInfo, conf *config.Co
 var tokenRegexp = regexp.MustCompile("token=[^&]+")
 
 func logFormatter(param gin.LogFormatterParams) string {
+	if param.ClientIP == "127.0.0.1" && param.Path == "/health" {
```

Comment:
Could you also check against ipv6 localhost `::1`. I'm offline for now, but will probably test this tomorrow.

Reply:
will do

Reply:
done